sql/mob_spawn_points.sql

```diff
@@ -117,904 +117,910 @@ INSERT INTO `mob_spawn_points` VALUES (16785462,'Land_Pugil','Land Pugil',17,-31
+INSERT INTO `mob_spawn_points` VALUES (16793690,'Coralline_Uragnite','Coralline Uragnite',8,-117.262,-0.280,-639.870,227);
+INSERT INTO `mob_spawn_points` VALUES (16793691,'Kraken','Kraken',9,-133.765,-0.384,-567.637,74);
+INSERT INTO `mob_spawn_points` VALUES (16793692,'Toucan','Toucan',14,-117.413,-0.245,-633.773,189);
+INSERT INTO `mob_spawn_points` VALUES (16793693,'Alraune','Alraune',13,-148.949,-3.127,-693.202,189);
+INSERT INTO `mob_spawn_points` VALUES (16793694,'Ignis_Fatuus','Ignis Fatuus',11,-164.371,-3.919,-682.986,86);
+INSERT INTO `mob_spawn_points` VALUES (16793695,'Viscous_Clot','Viscous Clot',10,-330.000,-0.001,-914.000,75);
+INSERT INTO `mob_spawn_points` VALUES (16793697,'Coastal_Opo-opo','Coastal Opo-opo',12,-158.243,-3.978,-600.555,31);
+INSERT INTO `mob_spawn_points` VALUES (16793696,'Coastal_Opo-opo','Coastal Opo-opo',12,-174.560,-3.796,-616.477,48);
+INSERT INTO `mob_spawn_points` VALUES (16793698,'Shankha','Shankha',17,-176.392,-3.776,-611.012,84);
+INSERT INTO `mob_spawn_points` VALUES (16793699,'Jagil','Jagil',7,-125.650,-0.340,-638.413,199);
+INSERT INTO `mob_spawn_points` VALUES (16793700,'Jagil','Jagil',7,-134.780,-1.145,-643.816,186);
+INSERT INTO `mob_spawn_points` VALUES (16793701,'Toucan','Toucan',14,-133.628,-1.156,-637.436,156);
+INSERT INTO `mob_spawn_points` VALUES (16793702,'Coastal_Opo-opo','Coastal Opo-opo',12,-187.453,-2.737,-573.818,171);
+INSERT INTO `mob_spawn_points` VALUES (16793703,'Alraune','Alraune',13,-213.348,-3.230,-553.583,185);
+INSERT INTO `mob_spawn_points` VALUES (16793704,'Alraune','Alraune',13,-206.285,-3.360,-547.666,246);
+INSERT INTO `mob_spawn_points` VALUES (16793705,'Toucan','Toucan',14,-223.342,-2.715,-533.340,2);
+INSERT INTO `mob_spawn_points` VALUES (16793706,'Jagil','Jagil',7,-285.000,0.001,-907.000,70);
+INSERT INTO `mob_spawn_points` VALUES (16793707,'Alraune','Alraune',13,-251.526,-3.930,-527.831,156);
+INSERT INTO `mob_spawn_points` VALUES (16793708,'Alraune','Alraune',13,-250.881,-3.837,-531.278,105);
+INSERT INTO `mob_spawn_points` VALUES (16793709,'Coastal_Opo-opo','Coastal Opo-opo',12,-268.025,-3.569,-526.613,241);
+INSERT INTO `mob_spawn_points` VALUES (16793710,'Ignis_Fatuus','Ignis Fatuus',11,-281.653,-3.996,-522.317,251);
+INSERT INTO `mob_spawn_points` VALUES (16793711,'Peerifool','Peerifool',18,-321.315,-2.726,-741.241,132);
+INSERT INTO `mob_spawn_points` VALUES (16793712,'Peerifool','Peerifool',18,-322.777,-2.768,-739.350,12);
+INSERT INTO `mob_spawn_points` VALUES (16793713,'Peerifool','Peerifool',18,-321.871,-2.696,-742.227,152);
+INSERT INTO `mob_spawn_points` VALUES (16793714,'Peerifool','Peerifool',18,-322.847,-2.623,-741.673,67);
+INSERT INTO `mob_spawn_points` VALUES (16793715,'Peerifool','Peerifool',18,-322.066,-2.615,-738.434,116);
+INSERT INTO `mob_spawn_points` VALUES (16793716,'Peerifool','Peerifool',18,-320.565,-2.797,-740.059,140);
```

Comment:
Others: These Peerifool are [spawned simultaneously during One Good Deed](https://ffxiclopedia.fandom.com/wiki/Peerifool). They're not a group of mobs someone pulled to a different location.